dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -200,6 +200,33 @@ def require_captcha
     render json: {key: site_key}
   end
 
+  # GET /api/v1/sections/<id>/code_review_groups
+  # Get all code review groups and their members for this section. Also include
+  # all unassigned followers. Format is:
+  # { groups: [
+  #   {unassigned: true, name: 'unassigned', members: [{follower_id: 1, name: 'student_name'},...]},
+  #   {id: <group-id>, name: 'group_name', members: [{follower_id: 2, name: 'student_name'},...]},
+  #   ...
+  # ]}
+  def code_review_groups
+    groups = CodeReviewGroup.where(section_id: @section.id)
+    groups_details = []
+    assigned_follower_ids = []
+    groups.each do |group|
+      members = []
+      group.members.each do |member|
+        members << {follower_id: member.follower_id, name: member.name}
+        assigned_follower_ids << member.follower_id
+      end
+      groups_details << {id: group.id, name: group.name, members: members}
+    end
+
+    unassigned_members = @section.followers.where.not(id: assigned_follower_ids)
+    unassigned_members = unassigned_members.map {|member| {follower_id: member.id, name: member.student_user.name}}
+    groups_details << {unassigned: true, name: 'unassigned', members: unassigned_members}
```

Comment:
Would it be an issue if a teacher creates a new group with the name `'unassigned'`? Should we prevent that on the front end or just not pass a name here?

Reply:
good point, the name isn't necessary so I'll remove it